event.go

```diff
@@ -665,8 +665,14 @@ func (e *Event) Interface(key string, i interface{}) *Event {
 }
 
 // Caller adds the file:line of the caller with the zerolog.CallerFieldName key.
```

Comment:
Can you document the new argument please?

Reply:
Modify now